content/blog/2020-02-10-introduction-to-machine-learning-with-the-tidyverse/index.Rmarkdown

```diff
@@ -0,0 +1,191 @@
+---
+title: 'Introduction to Machine Learning with the Tidyverse'
+date: '2020-02-10'
+slug: intro-to-ml-tidy
+categories: []
+tags: []
+author:
+  - alison
+  - garrett
+photo:
+  url: https://unsplash.com/photos/5Z9bgfRZLLE
+  author: Michael Lee
+description: >
+  Materials from our two-day rstudio::conf(2020) workshop on machine learning with the tidyverse and tidymodels are available. Here is your roadmap to the workshop!
+---
+
+
+> tl;dr: all workshop materials are available here: <br>
+> <i class="fab fa-github"></i> https://rstd.io/conf20-intro-ml <br>
+> `r emo::ji("link")` https://conf20-intro-ml.netlify.com/
+
+At [rstudio::conf(2020)](https://rstd.io/conf) in January, I was lucky to lead a new two-day workshop called "Introduction to Machine Learning with the Tidyverse." This workshop was designed for learners who are comfortable using R and the [tidyverse](https://www.tidyverse.org/), and curious to learn how to use these tools to do machine learning using the modern suite of R packages called [tidymodels](https://github.com/tidymodels). If you just read that last sentence and don't yet know the word "tidymodels" yet, it is a collection of modeling packages that work with the tidyverse. [Max Kuhn](https://twitter.com/topepos), the software engineer leading the tidymodels team and the author or maintainer of several R packages for predictive modeling including [caret](https://cran.r-project.org/web/packages/caret/caret.pdf), [Cubist](https://cran.r-project.org/web/packages/Cubist/index.html), [C50](https://cran.r-project.org/web/packages/C50/C50.pdf) and others, has offered an [Applied Machine Learning workshop](https://github.com/tidymodels/aml-training) for several years now, but we have not yet attempted to teach tidymodels to a beginner audience. Until now! `r emo::ji("tada")`
```

Comment:
Description of Max seems like a long interpolation in an already-long sentence. (Not saying he shouldn't be described, but it feels like it wants to be a sentence on its own.)